auto_nag/bugbug_utils.py

```diff
@@ -86,6 +52,7 @@ def get_bugs(self, date="today", bug_ids=[]):
         finally:
             Bugzilla.BUGZILLA_CHUNK_SIZE = old_CHUNK_SIZE
 
+        # Some consumer of this API needs the actual bugs so download them
```

Comment:
We shouldn't need this anymore.

Reply:
We can do a cleanup in a follow-up PR if you prefer, as there are a lot of things in the script we don't really need anymore.
Can you file an issue if you want to do it in a follow-up?

Reply:
Unfortunately we still need it because the scripts still access some of the bug fields, I have updated the comment to track which one of them need these data. Here is the detailed list below:

component.py:
    https://github.com/mozilla/relman-auto-nag/blob/b8c32c5c8825a211763eb8e303e0a3980228d54c/auto_nag/scripts/component.py#L91
defectenhancementtask.py:
    https://github.com/mozilla/relman-auto-nag/blob/b8c32c5c8825a211763eb8e303e0a3980228d54c/auto_nag/scripts/defectenhancementtask.py#L101
    https://github.com/mozilla/relman-auto-nag/blob/b8c32c5c8825a211763eb8e303e0a3980228d54c/auto_nag/scripts/defectenhancementtask.py#L111
regression.py:
    https://github.com/mozilla/relman-auto-nag/blob/b8c32c5c8825a211763eb8e303e0a3980228d54c/auto_nag/scripts/regression.py#L93
stepstoreproduce.py:
    https://github.com/mozilla/relman-auto-nag/blob/b8c32c5c8825a211763eb8e303e0a3980228d54c/auto_nag/scripts/stepstoreproduce.py#L74

Reply:
Yep, but we need only a few fields, like other normal scripts (by "normal" I mean not based on bugbu). So, like the others, we can simply get only those specific fields rather than the whole bug data.

Reply:
For example, https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/scripts/regression_without_regressed_by.py#L153.

Reply:
> For example, https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/scripts/regression_without_regressed_by.py#L153.

Can we add a similar lines in all the affected scripts? This should works without requiring heavy changes, right?

Reply:
I'm not totally sure about the extent of the changes, but it shouldn't be a lot.
The best way to do it would probably be to convert one script at a time to stop deriving from BugbugScript (and only share the code https://github.com/mozilla/relman-auto-nag/blob/5fa78296240c5f2e34421444768759482b95c911/auto_nag/bugbug_utils.py#L76-L98, which can be extracted in a separate function).